docs/guides/getting-started.md

```diff
@@ -55,4 +56,4 @@ resource "checkly_check_group" "first-group" {
 }
 ```
 
-5. Now run `$ terraform init` to setup a new project. Then use `$ terraform plan` followed by `$ terraform apply` to deploy your monitoring.
+6. Now run `$ terraform init` to setup a new project. Then use `$ terraform plan` followed by `$ terraform apply` to deploy your monitoring.
```

Comment:
@ianaya89 also this should say just 5, right?

Reply:
Mmm not sure if I get you, we have 6 steps.

Reply:
Mmm not sure if I get you, we have 6 steps.

Reply:
@ianaya89 forget about it, I didn't see the ellipsis in the diff. All good!